guides/common/modules/proc_installing-flatpak-applications-on-project-hosts.adoc

```diff
@@ -28,6 +28,15 @@ $ flatpak remote-add --authenticator-name=org.flatpak.Authenticator.Oci katello
 ----
 $ podman login _{foreman-example-com}_
 ----
++
+Alternatively,  you can configure the host to use container certificates for authentication.
```

Comment:
This one could aslo be moved like the proxy section to list podman login as an alternative.